### PR TITLE
Feature/horizontal chart

### PR DESCRIPTION
**Pull request by Bluesmile82**
- Add horizontal bar and horizontal stacked bar chart to docs
- Fix some warnings and refactor a bit the charts component

Note: Some details are missing like fixing the problem with the long texts in the x-axis. This could be solved with an ellipsis and a tooltip (react-popper) and maybe some legend for mobile versions

### CODE REVIEW

File: frontend/scripts/react-components/dashboard-element/dashboard-widget/dashboard-widget-config.js [hunk 1]
```diff
@@ -78,6 +78,32 @@ export default {
...
+    xKeys: {
+      bars: null
+    },
+    yKey: 'y',
+    xKeysAttributes: {
+      bars: {
+        interval: 'preserveStartEnd'
+      }
+    },
+    yAxis: {
```
Comment (sorodrigo) [inline frontend/scripts/react-components/dashboard-element/dashboard-widget/dashboard-widget-config.js]: This comes from the API in meta

File: frontend/scripts/react-components/chart/chart.component.jsx [hunk 2]
```diff
@@ -128,28 +115,8 @@ class Chart extends PureComponent {
...
-                    unit={unit || ''}
-                    unitFormat={unitFormat || (value => value)}
-                    fill="#555555"
-                  />
-                }
-                {...yAxis}
-              />
-            )}
-
+            {CustomXAxis({ config, data })}
```
Comment (sorodrigo) [inline frontend/scripts/react-components/chart/chart.component.jsx]: Why not mount it as a component?

Comment (Bluesmile82) [reply frontend/scripts/react-components/chart/chart.component.jsx]: I tried but the ComposedChart in recharts only accept some specific components. If you include something else it doesn't render it without a warning

**Comment by Bluesmile82** [comment]
Ready again! Let's fix the cursor in the chart style fixes PR

**Pull request merged**